### DISCUSSION TITLE
Reverse proxy - htaccess not working

### DISCUSSION THREAD

**Discussion by YohanRodriguezMDM** [YOURLS/YOURLS#3725] [Q&A]
Hi, 
I have a specific specific infrastructure, and I'm having an issue with my .htaccess.
I have a nginx reverse proxy on the front and my apache web server on the back (another server).
Files are in /var/www/yourls on the apache server.
I've used the .htaccess from the documentation but it's not redirecting the root.
My URL : [https://s.m2m.ag](https://s.m2m.ag)
Here is my .htaccess
```
# BEGIN YOURLS
  <IfModule mod_rewrite.c>
    RewriteEngine On
    RewriteBase /
    RewriteCond %{REQUEST_FILENAME} !-f
    RewriteCond %{REQUEST_FILENAME} !-d
    RewriteRule ^.*$ /yourls-loader.php [L]
    RewriteRule "(^|/)\.(?!well-known\/)" - [F]
  </IfModule>
# END YOURLS
```

Thanks for your help

**Comment by LeoColomb** [comment]
> I have a specific specific infrastructure

If you have a that specific infrastructure, we cannot recommend enough to reach out to your System Administrator.

**Reply by YohanRodriguezMDM** [reply]
Fun fact : I'm the sysadmin. The guy who built this infrastructure left the company.
I'll try to find a way to solve this. Thanks anyway :)
